packages/pytest-simcore/src/pytest_simcore/services_api_mocks_for_aiohttp_clients.py

```diff
@@ -31,6 +32,7 @@
     "pytest_simcore.aioresponses_mocker",
 ]
 
+_dummy_s3_url: str = "http://my-dummy-d3.com"
```

Comment:
Q: what will happen if I register this domain? Could you put a `.local` to end of it to avoid accidental tests being broken?